cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -458,9 +458,12 @@ spec:
                 sslCertificateRef: {}
                 port:
                   type: integer
-                  minimum: 1
+                  minimum: 0
```

Comment:
Whenever we make a change to the CRDs, there are several locations to update besides common.yaml:
- helm chart: cluster/charts/rook-ceph/templates
- integration tests: ceph_manifests.go
- upgrade manifest: we will worry about this when we get closer to v1.5. This change isn't critical for now in upgrades, even if we backport to 1.4.

Reply:
Done.